spec/spec_helper.rb

```diff
@@ -11,6 +11,20 @@
 
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
+  require "webmock/rspec"
+  WebMock.disable_net_connect!(allow_localhost: true)
+
+  config.before(:each) do
+    stub_request(:any, "https://brianiswu-open-brewery-db-v1.p.rapidapi.com/breweries/search").
+      with(
+        headers: {
+          "Accept" => "*/*",
+          "User-Agent" => "Ruby",
+        },
```

Comment:
Ignoring this for now, because I am getting conflicting messages from Hound and Rubocop about `Style/TrailingCommaInArguments: Avoid comma after the last parameter of a method call.`